testing/benchmarking/automated-benchmark/README.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        --set seldonDeployment.apiType="rest|grpc" \
</pre><p>If the value is defined as <code>A|B</code>  will it then create two workflows?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Exactly, you can define mutliple values and it will generate all the different combinations and run the equivallent number of jobs